### PR TITLE
Restore d3.js for timeline view

### LINKED ISSUE #12361: Timeline UI page is broken
Timeline.html file explicitly has:
```
    <script src="vendor/d3/d3-3.3.4.js"></script>
...
```
but it was removed in https://github.com/prestodb/presto/commit/e2aa9e139ea48430ad5446fcff786b7979aa8638

Therefore splits UI page no longer works

FYI: @raghavsethi

### PR DESCRIPTION
**Pull request by mayankgarg1990**
Resolves #12361

This file was removed in `e2aa9e1` but the timeline.html file still depends on it. Hence restoring it for now.

### CODE REVIEW

**Comment by rschlussel** [comment]
did you test that it fixes the problem? If so, I'll approve.  Not going to review the actual code, since I see it's just restoring code that had previously been deleted.

File: presto-main/src/main/resources/webapp/vendor/d3/d3-3.3.4.js [hunk 1]
```diff
@@ -0,0 +1,9294 @@
+!function() {
+  var d3 = {
+    version: "3.4.4"
```
Comment (wenleix) [inline presto-main/src/main/resources/webapp/vendor/d3/d3-3.3.4.js]: inconsistent version? 😃

**Comment by mayankgarg1990** [comment]
Screen Shot 2019-06-25 at 10 48 42 AM

The screenshot shows that the timeline view is now working

File: presto-main/src/main/resources/webapp/vendor/d3/d3-3.3.4.js (hunk 1 above)
Comment (mayankgarg1990) [reply presto-main/src/main/resources/webapp/vendor/d3/d3-3.3.4.js]: Copying the official `3.3.4` version

**Comment by rschlussel** [comment]
merging despite travis errors (travis is in a bad state now) since the web_ui build passed and the change only affects the web_ui.

**Pull request merged**